server/gameconnection.py

```diff
@@ -417,6 +349,21 @@ def send_message(self, message):
                 (teamkiller_id, victim_id, self.game.id, gametime)
             )
 
+    async def handle_ice_message(self, receiver_id, ice_msg):
+        peer = self.player_service.get_player(receiver_id)
```

Comment:
Make sure `receiver_id` is an int

```suggestion
        reveiver_id = int(receiver_id)
        peer = self.player_service.get_player(receiver_id)
```